### ISSUE TITLE
Wishlist: Support sqlite

### ISSUE THREAD

**Issue by bangzek** [slashbaseide/slashbase#30]
Cause that what I use

**Comment by paraswaykole** [comment]
Noted! That's what Slashbase too uses internally for saving app data. Will be add into the roadmap soon.

**Comment by DannyBen** [comment]
Yes please. Loved the quick docker installation and startup, was pleasantly surprised Mongo was there, but disappointed that SQLite was not. Can't wait for this to be added.
